testing/load-testing/README.md

```diff
@@ -0,0 +1,84 @@
+# K6
+
+K6 is a load testing tool that uses JavaScript and Go to fake multiple simultaneous requests to an API or Web Server.
+It gives instant information of the requests results and allows you to easily test the performance limits of your API.
+
+## Installing
+
+### Mac
+
+```
+brew tap loadimpact/k6
+brew install k6
+```
+
+### Docker
+
+```
+docker pull loadimpact/k6
+```
+
+### Linux
+
+[Download](https://github.com/loadimpact/k6/releases) and extract the latest version.
+
+## Usage
+
+### Script Usage
+
+After installing just execute `./path/to/k6 run --vus 5 -d 2s  script.js` if you are working in a Linux environment and you have extracted the tool from the .tar.gz or simply run `k6 run --vus 5 -d 2s  script.js` if you have are using Mac.
+
+The options are:
+
+- --vus: indicates how many virtual users you want to create
+- -d: test duration, 0 to run until cancelled (You need to specify the unit: 's', 'm' or 'h')
+
+You have different script examples in
+
+- [GET REQUEST](get.js)
+- [POST REQUEST](post.js)
+- [PUT REQUEST](put.js)
+
+### Before Testing
+
+Create a copy of this [document](https://docs.google.com/spreadsheets/d/1EUS4Livu-0rXkJnDx6oZjnl3mDA9PXlCfUSb_dlvPlU/edit#gid=0), where you shall put your testing results. This document shall remain with all the other project documentation.
```

Comment:
I still have no access to this. Would you move to the document to a `.md` file in this repository?